src/selfie_park/include/selfie_park/Search_server.h

```diff
@@ -45,10 +47,17 @@ class Search_server
   std::vector<Box> potential_free_places;
   Box first_free_place;
 
+  float length_of_parking_area_; // length of parking area, when this distance is covered service will be aborted
+  float max_distance_;
+  float current_distance_;
+  bool received_distance_;
```

Comment:
Nie rozumiem po co tworzona jest ta zmienna

Reply:
To stary śmieć